app/src/main/java/com/framgia/moneymanagement/screen/spending/SpendingAddActivity.java

```diff
@@ -0,0 +1,181 @@
+package com.framgia.moneymanagement.screen.spending;
+
+import android.app.DatePickerDialog;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.framgia.moneymanagement.R;
+import com.framgia.moneymanagement.data.model.Spending;
+import com.framgia.moneymanagement.data.repository.SpendingRepository;
+import com.framgia.moneymanagement.data.source.remote.SpendingRemoteDataSource;
+import com.framgia.moneymanagement.screen.income.IncomeAddActivity;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+public class SpendingAddActivity extends AppCompatActivity implements
+        SpendingAddContract.View {
+    private Spinner mSpinnerGroup;
+    private SpendingAddContract.Presenter mPresenter;
+    private EditText mEditTextNote, mEditTextAmount, mEditTextCalendar;
+    private TextView mTextViewTime, mTextViewTitile;
+    private Toolbar mToolbar;
+    private Calendar mCalendar;
+    private SimpleDateFormat mSimpleDateFormat = new SimpleDateFormat(SpendingKey.DATE_FORMAT);
+
+    public static Intent getIntent(Context context) {
+        return new Intent(context, SpendingAddActivity.class);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_spending_add);
+        initViews();
+    }
+
+    private void initViews() {
+        mPresenter = new SpendingAddPresenter(new SpendingRepository(
+                new SpendingRemoteDataSource(FirebaseDatabase.getInstance())), this);
+        mEditTextNote = findViewById(R.id.text_spending_note);
+        mEditTextAmount = findViewById(R.id.text_spending_amount);
+        mEditTextCalendar = findViewById(R.id.text_spending_calendar);
+        mSpinnerGroup = findViewById(R.id.spinner_spending_group);
+        mToolbar = findViewById(R.id.toolbar_spending_add);
+        mTextViewTime = findViewById(R.id.text_spending_calendar);
+        mTextViewTitile = findViewById(R.id.text_view_title_action_bar);
+        mCalendar = Calendar.getInstance();
+        mTextViewTime.setText(mSimpleDateFormat.format(mCalendar.getTime()));
+        initToolbar();
+        mEditTextCalendar.setOnClickListener(new View.OnClickListener() {
```

Comment:
nên implement hàm onClick cho dễ dang tracking 
```suggestion
        mEditTextCalendar.setOnClickListener(this)
```